src/Helpers_TEST.cc

```diff
@@ -25,21 +25,27 @@ using namespace fuel_tools;
 /////////////////////////////////////////////////
 TEST(HelpersTEST, UriToPathNoAuthority)
 {
+#ifdef WIN32
+  const std::string testStr1 = R"(localhost:8000\some\path)";
+  const std::string testStr2 = R"(localhost:8000\some\path\)";
+#else
+  const std::string testStr1 = R"(localhost:8000/some/path)";
+  const std::string testStr2 = R"(localhost:8000/some/path/)";
+#endif
+
   {
     common::URI uri{"http://localhost:8000"};
     EXPECT_EQ("localhost:8000", uriToPath(uri));
   }
 
   {
     common::URI uri{"http://localhost:8000/some/path"};
-    EXPECT_EQ(common::joinPaths("localhost:8000", "some", "path"),
-        uriToPath(uri));
+    EXPECT_EQ(testStr1, uriToPath(uri));
   }
 
   {
     common::URI uri{"http://localhost:8000/some/path/"};
-    EXPECT_EQ(common::separator(common::joinPaths("localhost:8000", "some",
-        "path")), uriToPath(uri));
+    EXPECT_EQ(testStr2, uriToPath(uri));
```

Comment:
I think If this test is changed, the tests below with `UriToPathHasAuthority` should change as well

Reply:
ee7cae2